ansible/configs/smart-management/sample_vars_osp.yml

```diff
@@ -10,17 +10,93 @@ student_name: student
 student_password: "r3dh4t1!"
 lab_user: devops
 
+setup-lab: false                      # Whether or not to setup tower and satellite extra environment
+setup-sync: true
+
+insights_auth:
+  username: luke-skywalker             # Replace with Red Hat Insights username
+  password: SithLordDads               # Replace with Red Hat Insights password
+
 #### Tower variables ############
 tower_version: 3.7.1-1
 tower_hostname: "tower.{{guid}}.{{osp_cluster_dns_zone}}"
 software_to_deploy: tower              # tower install playbook from software directory
-
+devops_password_tower: "r3dh4t1!"
```

Comment:
please make this a variable, no default password in the code, or

```
 {{ lookup('password', '/dev/null length=15 chars=ascii_letters') }}
```

Reply:
Good catch